bin/cellranger_mtx_to_h5ad.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+import scanpy as sc
+import argparse
+
+def mtx_to_adata( mtx_dir: str, verbose: bool = False ):
+
+    if verbose:
+        print("Reading in {}".format(mtx_dir))
+
+    adata = sc.read_10x_mtx(mtx_dir)
+
+    return adata
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Converts mtx output to h5ad.")
+
+    parser.add_argument("-m", "--mtx",     dest="mtx",     help="Path to mtx directory."                 )
+    parser.add_argument("-v", "--verbose", dest="verbose", help="Toggle verbose messages", default=False )
+    parser.add_argument("-o", "--out",     dest="out",     help="Output path."                           )
+
+    args = vars(parser.parse_args())
+
+    adata = mtx_to_adata(args["mtx"], verbose=args["verbose"])
+
+    adata.write_h5ad(args["out"])
```

Comment:
```suggestion
    adata.write_h5ad(args["out"], compression="gzip")
```

Or `compression="lzf"` if you know it will be available.

Reply:
`lzf` is much faster, though files will be a bit larger. It will always be available with h5py, but might not for other hdf5 installs.

Reply:
Thanks for the review. I will add it rightaway.